recipes/einsteinpy/meta.yaml

```
@@ -0,0 +1,57 @@
+{% set name = "einsteinpy" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  md5: 8a8815ae9341f4b544c496a697342892
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+  skip: True
```

Currently this is never built as it's always skipped, I'm not sure what you're trying to do here?

Unrelated, this looks like it could be a [noarch package](https://docs.conda.io/projects/conda-build/en/latest/source/resources/define-metadata.html?highlight=selectors#architecture-independent-packages):
```suggestion
  noarch: python
```